static/js/modules/send-message.js

```diff
@@ -6,10 +6,6 @@ var _ = require('underscore'),
 
   'use strict';
 
-  var translateFn = function(key) {
```

Comment:
Erm IDK. That's a potential point. I did test it with that function logging out to see if it was ever called and couldn't do it, but that doesn't cover all cases obviously. I'll do a little investigation…